dotcom-rendering/src/model/buildLightboxImages.ts

```diff
@@ -96,33 +65,19 @@ const buildLightboxImage = (
 	const width = parseInt(masterImage.fields.width, 10);
 	const height = parseInt(masterImage.fields.height, 10);
 
-	const orientation = width >= height ? 'landscape' : 'portrait';
-	if (!isLightboxable(element, orientation)) return;
+	if (!isLightboxable(width, height)) return;
 
 	return {
 		masterUrl: masterImage.url,
-		position: element.position,
 		width,
 		height,
 		elementId: element.elementId,
-		alt:
-			element._type ===
-			'model.dotcomrendering.pageElements.ImageBlockElement'
-				? element.data.alt
-				: element.alt,
-		credit: element.lightbox?.credit,
-		caption: element.lightbox?.caption,
+		alt: isImage(element) ? element.data.alt : element.alt,
```

Comment:
Minor - `isImage(element)` is called a few times in this function could it be assigned to a const once?